lint/backend.py

```diff
@@ -41,7 +41,8 @@ def lint_view(view, hit_time, callback):
 
     all_errors = chain.from_iterable(results)
 
-    callback(view, list(all_errors), hit_time)
+    sublime.set_timeout_async(
+        lambda: callback(view, list(all_errors), hit_time))
```

Comment:
I generally propose using `functools.partial` when a function is only invoked with arguments and no other logic is necessary. Lambdas are evalualted lazily which *can* in some occasions, particularly within loops, produce unexpected results.